docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -1,10 +1,7 @@
-:tutorial: tutorial
-:gcp: GCP
-
 [[monitor-gcp]]
```

Comment:
There's a missing {gcp} removal on this page somewhere.

```
<p>Go to the <span class="strong strong"><strong>Logs Router</strong></span> page to configure {gcp} to export logs to a Pub/Sub
```

https://observability-docs_bk_3673.docs-preview.app.elstc.co/guide/en/observability/master/gcp-dataflow.html